integration/spark/src/main/java/openlineage/spark/agent/lifecycle/RddExecutionContext.java

```diff
@@ -190,19 +200,16 @@ protected ZonedDateTime toZonedTime(long time) {
     return ZonedDateTime.ofInstant(i, ZoneOffset.UTC);
```

Comment:
I think we should stop reporting the timestamps in UTC- the processing jobs are often running in locale-specific timezones and that is useful information to collect.